src/main/java/com/tradeshfit/model/dao/JdbcMessagesDAO.java

```diff
@@ -0,0 +1,78 @@
+package com.tradeshfit.model.dao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import javax.sql.DataSource;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class JdbcMessagesDAO implements MessagesDAO {
+
+    private DataSource dataSource;
+    public JdbcMessagesDAO(){
+
+    }
+    @Autowired
+    public JdbcMessagesDAO(DataSource dataSource){
+        this.dataSource = dataSource;
+    }
+
+    public void insert(String message, Date created) {
+        String insertSql = "INSERT INTO recent (message, created) VALUES (?, ?)";
+        Connection connection = null;
+        try {
+            connection = dataSource.getConnection();
+            PreparedStatement preparedStatement = connection.prepareStatement(insertSql);
+
+            preparedStatement.setString(1, message);
+            preparedStatement.setTimestamp(2, new Timestamp(created.getTime()));
+            preparedStatement.executeUpdate();
+            preparedStatement.close();
+        } catch (SQLException e){
+            throw new RuntimeException(e);
+        } finally {
+            if(connection != null){
```

Comment:
This is better to do with "try-with-resources" in java 7. That will make sure that the connection is closed when the try/catch completes.

https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html